src/main/flight/mixer.c

```diff
@@ -306,6 +306,19 @@ static float mixerCollectiveCorrection(float SC)
     return SC;
 }
 
+static float mixerCollectiveScale(float SC, float SR, float SP)
+{
+    float beta;
+    if (SC > 0) {
+        beta = mixerConfig()->collective_scale_pos / 100.0f;
+    } else {
+        beta = mixerConfig()->collective_scale_neg / 100.0f;
+    }
+    float scale = 1 - beta * (SR * SR + SP * SP);
+    scale = constrainf(scale, 0, 1.0f);
```

Comment:
Slightly inconsistent. constrainf takes three float arguments. You are providing an int and a float.
Maybe just use two ints here. Or two floats. Whichever you prefer.

Reply:
Done